apps/coscrad-frontend/src/components/resources/bibliographic-citations/book-bibliographic-citation-detail.full-view.presenter.tsx

```diff
@@ -29,7 +32,15 @@ export const BookBibliographicCitationDetailFullViewPresenter = ({
     };
 
     // Temporary workaround until `name` is on IBaseViewModel
-    const name = title;
+    const name: IMultilingualText = {
```

Comment:
Can't we do this in the query service on the backend?

Reply:
This is already on the interface and in the query response from the back-end. I have removed this in favor of using the incoming props.